packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -160,9 +174,7 @@ export function buildClientOptions(): LanguageClientOptions {
     synchronize: {
       configurationSection: 'apex',
       fileEvents: [
-        vscode.workspace.createFileSystemWatcher('**/*.cls'), // Apex classes
-        vscode.workspace.createFileSystemWatcher('**/*.trigger'), // Apex triggers
-        vscode.workspace.createFileSystemWatcher('**/*.apex'), // Apex anonymous scripts
+        vscode.workspace.createFileSystemWatcher('**/*.{cls,trigger,apex}'), // Apex classes
```

Comment:
Reduce the number of file watchers  by combining apex artifacts into a single watcher